apps/generic_terminal/generic_unix_shell.py

```diff
@@ -1,11 +1,31 @@
+import csv
+from pathlib import Path
+
 from talon import Context, Module, actions
 
+from ...code.user_settings import get_list_from_csv
+
 ctx = Context()
 mod = Module()
 ctx.matches = r"""
 tag: user.generic_unix_shell
 """
 
+path = Path(__file__).parents[0]
+with open(path / "default_unix_utilities.csv") as f:
+    # get_list_from_csv() expects the columns in the reverse order
+    default_unix_utilities = {spoken: command for command, spoken in csv.reader(f)}
```

Comment:
hmm i wonder if there's a way to avoid reading this every time, because after the first time it will just ignore it in favour of user csv, right? maybe `get_list_from_csv` could take a callback that it only calls if it can't find user one?  Idk I guess a callback maybe isn't very pythonic 🤔.  any thoughts @auscompgeek

Reply:
I further simplified this by using a static dict to initialize the CSV. This keeps it from loading the default CSV on every launch.